Impl save time ldap user

BoolPurist opened the pull request:
Added opt in feature to creation date for LDAP user
To preserve the backwards compatibility with earlier versions, this features must be opted in.

You can opt in by the following steps:

1. Apply the changes from the file docker/bootstrap_lidfs/02_scheme_ldap_attr.ldif
2. Set the field value "ldap_add_created_at" to `true` within the configuration file.
3. Add the value "ldapAttributes" to the array value "objectclass_common" within the configuration file.

BoolPurist committed:
Added opt in feature to creation date for LDAP user

To preserve the backwards compatibility with earlier versions, this features must be opted in.

You can opt in by the following steps:

1. Apply the changes from the file docker/bootstrap_lidfs/02_scheme_ldap_attr.ldif
2. Set the field value "ldap_add_created_at" to `true` within the configuration file.
3. Add the value "ldapAttributes" to the array value "objectclass_common" within the configuration file.

BoolPurist committed:
Merge branch 'main' into impl_save_time_ldap_user

BoolPurist commented:
Questions to @dwgnr: 

1. Do like the name  "ldapAttributes" for objectclass under which "created_at" attribute is held  ?
2. Do agree with the opt in approach ? Allowing to use this tool without having this new attribute in the scheme.

dwgnr commented on usermgmt_lib/src/ldap.rs:
```diff
@@ -115,26 +116,33 @@ where
+                ("sshPublicKey", hashset! {pubkey}),
+                ("loginShell", hashset! {config.login_shell.as_str()}),
+            ];
+            let created_at = Utc::now().to_rfc3339();
```
please move this down

dwgnr reviewed (approved):
as discussed

BoolPurist committed:
Docs: Documented time stamp representation

Fix: UTC time stamp creation is now only done
if add time stamp feature is toggled on
